pkg/cmd/cli/describe/describer.go

```diff
@@ -980,7 +980,7 @@ func DescribePolicy(policy *authorizationapi.Policy) (string, error) {
 	})
 }
 
-const policyRuleHeadings = "Verbs\tResources\tResource Names\tNon-Resource URLs\tExtension"
+const policyRuleHeadings = "Verbs\tNon-Resource URLs\tExtension\tResource Names\tAPI Groups\tResources"
```

Comment:
This reordering looks a little strange, but the list of `Resources` is very large, so putting it at the end helps keep the other columns more orderly.